src/components/modebar/index.js

```diff
@@ -97,18 +121,35 @@ proto.createButton = function (config) {
     button.setAttribute('rel', 'tooltip');
     button.className = 'modebar-btn';
 
-    if (config.attr !== undefined) button.setAttribute('data-attr', config.attr);
-    if (config.val !== undefined) button.setAttribute('data-val', config.val);
-    button.setAttribute('data-title', config.title);
-    button.setAttribute('data-gravity', config.gravity || 'n');
-    button.addEventListener('click', function () {
-            config.click.apply(_this, arguments);
+    var title = config.title;
+    if(title !== null && title !== false && title !== '') {
```

Comment:
good point. Could also take anything falsey except `undefined` as "no title," ie `if(title || title === undefined)` - actually, what do we want to happen if title is `0`? I can imagine people getting confused results from that, like if someone makes a set of buttons that they want to label 0, 1, 2, 3, but the first one comes out labeled "setParamCount0" or something... strange, but we might as well plan for it, something like:
```javascript
var title = config.title;
if(title === undefined) title = config.name;
if(title || title === 0) button.setAttribute('data-title', title);
```